docker/cytoscape/DockerFile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+# PARAMETERS
+ENV CYTOSCAPE_VERSION 3.8.2
+
+# CHANGE USER
+USER root
+
+# INSTALL JAVA
+RUN apt-get update && apt-get -y install default-jdk libxcursor1 xvfb supervisor wget x11vnc
+RUN wget https://github.com/cytoscape/cytoscape/releases/download/3.8.2/cytoscape-unix-3.8.2.tar.gz
+RUN tar xf cytoscape-unix-3.8.2.tar.gz && rm cytoscape-unix-3.8.2.tar.gz
+
+# Set JAVA_HOME From sudo update-alternatives --config java
+RUN echo 'JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"' >> /etc/environment
+
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+
+ENV PATH /opt/conda/bin:$PATH
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda clean --all && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    conda init bash
+
+COPY workflow/minimal_env.yml ./docker/
+RUN conda env create -f docker/minimal_env.yml
```

Comment:
When is this environment activated?

Reply:
In `docker/cytoscape/tests/connect.sh` . This is a proof of concept script that calls `connect_cytoscape.py` which establishes a connection to Cytoscape in a headless mode